### PR TITLE
about me #251

### PR DESCRIPTION
**Pull request by jiaqianjing**

### CODE REVIEW

File: jekyll/_contributors/jiaqianjing.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: jiaqianjing
+site: https://github.com/jiaqianjing
+avatar: /assets/contributors/jiaqianjing/avatar.webp
+bio: Native Clouder or NLPer orzzzzzz. Be interested in anything.
+email: jiaqianjing@gmail.com
+twitter: 
+---
+
+仙友你好!
```
Comment (lijiarui) [inline jekyll/_contributors/jiaqianjing.md]: Please link your PR content here, thanks!

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: jekyll/_contributors/jiaqianjing.md (hunk 1 above)
Comment (jiaqianjing) [reply jekyll/_contributors/jiaqianjing.md]: ok

**Comment by lijiarui** [comment]
Please make github adction green.

**Comment by lijiarui** [comment]
Btw, please accept the invitation as  https://github.com/wechaty/python-wechaty/pull/251 requested, then Github action will run automatically for you.

**Comment by lijiarui** [comment]
@jiaqianjing 

Please accept the invitation as  https://github.com/wechaty/python-wechaty/pull/251#issuecomment-907056950 requested, then Github action will run automatically for you.

> You'are invited jiaqianjing to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

**Comment by jiaqianjing** [comment]
> > You'are invited jiaqianjing to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

Please tell me what I have not finished.

**Comment by lijiarui** [comment]
Please visit https://github.com/wechaty to accept the invitation.

**Review by lijiarui — APPROVED** [review]

**Comment by jiaqianjing** [comment]
> Please visit https://github.com/wechaty to accept the invitation.

Done

**Pull request merged**